src/Microsoft.Azure.NotificationHubs.DotNetCore.Tests/ManagementApiE2ETests.cs

```diff
@@ -99,6 +123,44 @@ public void ExecuteCreateGetAndDeleteNotificationHubMethods_ShouldCreateGetOrDel
             CleanUp();
         }
 
+        [Fact]
+        public void SubmitAndGetNotificationHubJob_ShouldReceiveCorrectJobs()
```

Comment:
Why not make this an async test if the underlying code is async, for example `public async Task SubmitAndGetNotificationHubJob_ShouldReceiveCorrectJobs()`?